controllers/vdoconfig_controller.go

```diff
@@ -867,6 +868,7 @@ nodeLoop:
 			continue nodeLoop
 		}
 
+		allNodesReady = false
```

Comment:
can we assign this boolean inside the if loop for taints where we set the cpi status as configuring

Reply:
this will ensure that we are only concerned about nodes that are intended by the user to be managed by cloud provider